src/hooks/useHandleAxiosError.js

```diff
@@ -0,0 +1,21 @@
+import { useIntl } from 'react-intl';
+
+export default function useHandleAxiosError() {
+  const intl = useIntl();
+  const defaultError = intl.formatMessage({
+    id: 'ERROR_FORMATTING_JSON',
+  });
+
+  return function handleAxiosError(axiosError) {
+    let error = defaultError;
+
+    if (axiosError.response?.data?.message) {
+      error = axiosError.response.data.message;
+    } else if (axiosError.toJSON) {
+      error = axiosError.toJSON().message;
+    } else if (axiosError.message) {
+      error = axiosError.message;
+    }
+    return Promise.reject(error);
+  };
```

Comment:
This is largely a copy of `formatError` in `useMutate`
https://github.com/WildMeOrg/codex-frontend/blob/d558ff2fd6fed3b5b9980b8080814ce586562b73/src/hooks/useMutate.js#L12